Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM linode/apl-tools:v2.7.0 as ci
+FROM linode/apl-tools:v2.7.1 as ci
```

Comment:
you are about to release  [TOOLS][MINOR] however you update here the patch version

Reply:
Yes, I used the patch release to test the new apl-tools image. This one is created by running the branch via gh-actions.
After the pr is merged I wanted to create a new pr with the update for the v2.8.0 image, as Ani mentioned.